### ISSUE TITLE
TooManyStatementsPerLineCheck creates false positive with type alias

### ISSUE THREAD

**Issue by guwirth** [wenns/sonar-cxx#656]
TooManyStatementsPerLineCheck creates false positive with type alias (see also #552).

```Java
template <typename RBM, typename Trainer>
void update_normal(RBM& rbm, Trainer& t) {
    using rbm_t = RBM; // <== TooManyStatementsPerLineCheck: Statements should be on separate lines    
} 
```
